lib/whoami.js

```diff
@@ -2,14 +2,31 @@
 
 // local modules
 
-const httpUtils = require('./utils/http');
 const jsonUtils = require('./utils/json');
+const pull = require('./pull');
+const urlUtils = require('./utils/url');
 const values = require('./values');
 
 // this module
 
 module.exports = {
 
+  getAuthentication (options) {
+    options = options || {};
+    const userConfigFile = values.userConfigFile(options.userConfigDir);
+    const origin = urlUtils.getOrigin(options.scope || '');
+    return jsonUtils.loadJsonObject(userConfigFile)
+      .catch(() => [])
```

Comment:
I think this will swallow any errors thrown by `loadJsonObject`, would it be worth outputting a message to the console or log?